empathylink.lic

```diff
@@ -0,0 +1,118 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#empathylink
```

Comment:
How does this script differ from `healer` or `smarttransfer`?

Reply:
Smarttransfer and healer are not setup to work in combat. This script creates links with people in your hunting-buddies list to link with, with Hodierna, to keep the buddies wound free.